services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/links/LinkSpec.groovy

```diff
@@ -353,6 +353,10 @@ class LinkSpec extends BaseSpecification {
         assert flow2Path == switchPair.paths.first()
 
         and: "Delete link props from all links of alternative paths to allow rerouting flows"
+        def allLinkProps = northbound.getAllLinkProps()
+        //DELETE /link/props deletes props for both directions, even if only one direction specified
+        northbound.deleteLinkProps([allLinkProps.first()])
+        assert northbound.getAllLinkProps().size() == allLinkProps.size() - 2
         northbound.deleteLinkProps(northbound.getAllLinkProps())
```

Comment:
Why you run the delete operation twice here?
Also, there are plenty of similar places where the same can be verified in LinkPropertiesSpec, why add this strange step in LinkSpec. 
Also, I'm wondering whether the `update` operation is updating both parts of link prop.

Reply:
I lost my train of thoughts(
verification was moved to the `LinkPropertiesSpec.groovy` file

Reply:
> I'm wondering whether the update operation is updating both parts of link prop.

I think so.
if you update one direction then the reversed direction will be updated too

Reply:
> I think so.
should be tested explicitly

Reply:
added